content/blog/2015-04-23-prometheus-posts-around-the-web.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Prometheus posts around the web"
+created_at: 2015-04-23
+kind: article
+author: Brian Brazil
+---
+
+It has been almost 3 months since Prometheus version [0.10.0 was publicly
+announced](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud).
+
+SoundCloud's post remains the best overview of the key components of
```

Comment:
It took me a second to realize that the first link is actually the post on developers.soundcloud.com. Maybe move the link around "SoundCloud's post" instead?